Api/Client.php

```diff
@@ -310,6 +307,75 @@ private function removeLastLineFromFiles($files = null, $noOfLinesToRemove = 1)
 
     /**
      *
+     * @param array $lines
+     * @param int   $noOfLinesToRemove
+     *
+     * @return array
+     */
+    private function removeLastLines(array $lines, $noOfLinesToRemove = 1)
+    {
+        $removeFrom = count($lines) - $noOfLinesToRemove;
+
+        for( $i = $removeFrom; $i < count($lines); $i++ ) {
+            unset($lines[$i]);
+        }
+        return $lines;
+    }
+
+    /**
+     * @param array $lines
+     * @param string $separator
+     * @param string $enclosure
+     *
+     * @return array
+     */
+    private function fixDate(array $lines, $separator = ',', $enclosure = '"')
+    {
+        foreach ($lines as $key => $line) {
+            $columns = str_getcsv($line, $separator );
+            $isChanged = false;
+            foreach ($columns as $columnKey => $column)
+            {
+                if (preg_match('/^[0-9]{1,2}\/[0-9]{1,2}\/[0-9]{4}$/', $column))
```

Comment:
You can be sure that this is a M/dd/YYYY. To prevent o have more replacements in the file you can make it more specific for this kind of date. Maybe more like ^[0-9]{1,2}\/[0-3][0-9]\/20[1-3][0-9]$